### PR TITLE
cmd/syncthing: Improve logged device information on startup

### PR DESCRIPTION
**Pull request by imsodin**

### CODE REVIEW

**Comment by calmh** [comment]
Why not?

**Comment by imsodin** [comment]
Because to me it seems weird to print the connection information to ourselves, which is anyway always the default `[dynamic]`. And we already print our own ID during startup before this.

**Comment by AudriusButkevicius** [comment]
I agree with this

**Comment by calmh** [comment]
Fair enough on the addresses, but the name is sometimes useful. I'd rather leave it in.

**Comment by imsodin** [comment]
I didn't expect so much discussion, I just wanted to get rid of this from my git workspace :)

Next try: Adding the name to the earlier "My ID" message, skipping the local device and not skipping devices without a name (IDs and connections are still relevant).

**Comment by calmh** [comment]
It's a trivial change, so we can discuss it forever. You should have just snuck it in among something more complex.

For example, I wonder if there isn't someone out there parsing the "My ID" line at startup to get the device ID...

File: cmd/syncthing/main.go [hunk 1]
```diff
@@ -828,9 +828,11 @@ func syncthingMain(runtimeOptions RuntimeOptions) {
 		pprof.StartCPUProfile(f)
 	}
 
+	myDev, _ := cfg.Device(myID)
+	l.Infof(`My device is "%v"`, myDev.Name)
```
Comment (calmh) [inline cmd/syncthing/main.go]:
```suggestion
	l.Infof(`My name is "%v"`, myDev.Name)
```

Comment (imsodin) [reply cmd/syncthing/main.go]: Fancy :)

**Comment by calmh** [comment]
I fail to find more things to complain about or discuss here.

**Comment by calmh** [comment]
Oh! Except the commit subject is now wrong of course. Caveat merger.

**Pull request merged**